versions.tf

```diff
@@ -20,11 +20,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 4.0"
-    }
-    google-beta = {
-      source  = "hashicorp/google-beta"
-      version = "~> 4.0"
+      version = "~> 5.0"
```

Comment:
I would leave this as `>=`. We don't want our modules to be overly restrictive on the available provider versions (it's led to headaches in the past). Also the fact that nothing else is changing means we probably don't need to make this PR. We can leave it as `>= 4.0` and it'll work just fine, no need to make this overly prescriptive